### PR TITLE
Use min-height of textarea when computing auto-resize after deleting a char

### LINKED ISSUE #482: Text in chat input invisible in landscape mode (mobile)
https://streamable.com/h6dc

### PR DESCRIPTION
**Pull request by astorije**
Fixes #482.

This fixes a bug that was breaking the textarea on Android.
@williamboman, can you give it a try?

### CODE REVIEW

File: client/css/style.css [hunk 1]
```diff
@@ -1260,6 +1260,7 @@ button {
 }
 
 #form #input {
+	overflow-y: hidden;
```
Comment (maxpoulin64) [inline client/css/style.css]: This will break scrolling when it hits 6-7 lines, as we stop growing at this point.

File: client/js/lounge.js [hunk 2]
```diff
@@ -628,12 +628,23 @@ $(function() {
...
 	var input = $("#input")
 		.history()
 		.on("input keyup", function() {
 			var style = window.getComputedStyle(this);
-			this.style.height = "0px";
-			this.offsetHeight; // force reflow
+
+			if (!initialInputHeight) {
+				// When first resizing the textarea, store its initial height for later
+				initialInputHeight = this.style.height;
```
Comment (maxpoulin64) [inline client/js/lounge.js]: There's no need to store it this way. You already have it in `style.minHeight` which contains the minimum height which is normally what prevents the 0px from applying in non-retarded browsers. So we can just use this one instead.

File: client/css/style.css (hunk 1 above)
Comment (astorije) [inline client/css/style.css]: Right, good point, I'll remove this.

File: client/js/lounge.js (hunk 2 above)
Comment (astorije) [inline client/js/lounge.js]: Actually, there is a height, not a min-height at the moment in https://github.com/thelounge/lounge/blob/ebab910e0013b15ff0bcc754b2003fb40e7d2298/client/css/style.css#L1262-L1275, so I need to add it. Will do.

**Comment by williamboman** [comment]
Functionality-wise works fine, :+1:.

**Comment by astorije** [comment]
@maxpoulin64 is 👍 (but didn't comment with 👍 itself, just the second-review label). All other maintainers are away at the moment, but @williamboman did test this successfully, so merging.

**Pull request merged**